AndroidApp/README.md

```
@@ -1,5 +1,11 @@
 # Androidアプリ
 
+### 開発環境
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)